README.md

```diff
@@ -2,76 +2,91 @@
-extend = mixer.mixin; // same as above.
+ // the regular Object.assign function
+var extend = mixer();
+// OR
+extend = mixer.mixin;
 
-extend({}, { a: 1 }, { b: 2 }); // { a: 1, b: 2 }
+extend({}, { a: 1 }, { b: 2 });
+// { a: 1, b: 2 }
 ```
 
-Mixin functions only.
+### Mixin functions only.
 ```js
-var functionMixer = mixer({ // assigns own functions only
+// assigns own functions only
+var functionMixer = mixer({
   filter: function (val) { return typeof val === 'function' ; }
 });
-functionMixer = mixer.mixinFunctions; // same as above
+// OR
+functionMixer = mixer.mixinFunctions;
 
-functionMixer({}, { a: "x" },  { b: function(){} }); // { b() }
+functionMixer({}, { a: "x" },  { b: function(){} });
+// { b() }
 ```
 
-Mixin functions including prototype chain.
+### Mixin functions including prototype chain.
 ```js
-var chainFunctionMixer = mixer({ // assigns functions only, but traverse through the protorype chain
+// assigns functions only, but traverse through the protorype chain
+var chainFunctionMixer = mixer({
   filter: function (val) { return typeof val === 'function' ; },
   chain: true
 });
-chainFunctionMixer = mixer.mixinChainFunctions; // same as above
+// OR
+chainFunctionMixer = mixer.mixinChainFunctions;
 
-chainFunctionMixer({}, new EventEmitter()); // { on(), off(), emit(), ... }
+chainFunctionMixer({}, new EventEmitter());
+// { on(), off(), emit(), ... }
 ```
 
-Deep merge data of any number of objects to a new object.
+### Deep merge data of any number of objects to a new object.
 ```js
-var mergeDeep = mixer({ // deep merge own properties.
+// deep merge own properties
+var mergeDeep = mixer({
   filter: function (val) { return typeof val !== 'function'; },
   deep: true
 });
-mergeDeep = mixer.merge; // same as above
+// OR
+mergeDeep = mixer.merge;
 
-mergeDeep({ url: { host: "example.com" } }, { url: { port: 81 } }); // { url: { host: "example.com", port: 81 } }
+mergeDeep({ url: { host: "example.com" } }, { url: { port: 81 } });
+// { url: { host: "example.com", port: 81 } }
 ```
 
-Deep merge non functions to a new object, including prototype chain.
+### Deep merge non functions to a new object, including prototype chain.
 ```js
-var mergeChainData =  mixer({ // deeply merges data properties, traversing through prototype chain
+// deeply merges data properties, traversing through prototype chain
+var mergeChainData =  mixer({
   filter: function (val) { return typeof val !== 'function'; },
   deep: true,
   chain: true
 });
-mergeChainData = mixer.mergeChainNonFunctions; //same as above
+// OR
+mergeChainData = mixer.mergeChainNonFunctions;
 
 EventEmitter.prototype.hello = "world";
-mergeChainData(new EventEmitter()); // { hello: "world" }
+mergeChainData(new EventEmitter());
+// { hello: "world" }
 ```
 
-# TODO
+## TODO
 * Tests
 * Travis
 * More `mixer.FUNCTION` functions
 
-# Want to contribute?
-It is Open Open Source. Whoever sends a PR, which gets accepted, receives the write permissions.
```

Comment:
May the following wording  would sound better?
`This is Open-Open Source project (yes, double open). Meaning that whoever sends a PR, which gets accepted, receives the write permissions.`